packages/side-nav/src/SideNavItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { MenuItem, MenuItemProps } from '@leafygreen-ui/menu';
+import { LEFT_RIGHT_OFFSET } from './styles';
+
+// style overrides for menu item
+const containerStyleOverrides = css`
+  padding: 10px ${LEFT_RIGHT_OFFSET}px 10px ${LEFT_RIGHT_OFFSET}px;
+  min-height: 0;
+  border-radius: 5px;
+
+  &:before {
+    content: normal;
```

Comment:
Is this to unset the left bar in the MenuItem? I think `initial` is preferable, if so.

Reply:
It is meant to unset the pseudo-elements, but MDN specifically states that this is the intended usage of `normal` (see [refs here](https://developer.mozilla.org/en-US/docs/Web/CSS/content#values))

Reply:
This is more of a preference, but usually I use `initial` rather than the default value to clearly communicate the intent of the rule to set it to its default value. Without looking at documentation, that's unclear here unless you're intimately familiar with the content property.

That said, I don't feel strongly about this. Either value should function the same here.

Reply:
This will be addressed as part of removing MenuItem as a dependency for SideNavItem